pkgs/development/python-modules/monai/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, numpy
+, pytorch
+}:
+
+buildPythonPackage rec {
+  pname = "monai";
+  version = "0.8.0";
+
+  disabled = pythonOlder "3.6";
+
+  format = "wheel";
+
+  # building from source with Ninja attempts to access network, so use PyPI wheel instead
```

Comment:
Can we patch the ninja files to prevent that?

Reply:
Now built from source instead of wheel.